<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="P3BLAP-9388WRB" failures="1" tests="3" name="com.herokuapp.theinternet.jsalerttests.JSAlertTests" time="46.165" errors="0" timestamp="13 Jul 2021 06:15:35 GMT" skipped="0">
  <testcase classname="com.herokuapp.theinternet.jsalerttests.JSAlertTests" name="jsAlert" time="18.191"/>
  <testcase classname="com.herokuapp.theinternet.jsalerttests.JSAlertTests" name="jsPrompt" time="14.949"/>
  <testcase classname="com.herokuapp.theinternet.jsalerttests.JSAlertTests" name="jsConfirm" time="13.025">
    <failure message="The following asserts failed:
	Alert message is not expected. 
Should be &#039;I am a JS Confirm&#039;, but it is &#039;I am a JS Confirm&#039; expected [true] but found [false],
	Result message is not expected. 
Should be &#039;You clicked: Cancel&#039;, but it is &#039;You clicked: Cancel&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Alert message is not expected. 
Should be 'I am a JS Confirm', but it is 'I am a JS Confirm' expected [true] but found [false],
Result message is not expected. 
Should be 'You clicked: Cancel', but it is 'You clicked: Cancel' expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.herokuapp.theinternet.jsalerttests.JSAlertTests.jsConfirm(JSAlertTests.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- jsConfirm -->
</testsuite> <!-- com.herokuapp.theinternet.jsalerttests.JSAlertTests -->
